flow/designs/gf12/coyote/rules-base.json

```diff
@@ -67,22 +67,22 @@
         },
         {
             "field": "finish__timing__setup__ws",
-            "value": -4.19,
+            "value": -22.01,
```

Comment:
This makes the results worse rather than better.   Isn't this the design abk was concerned about?

Reply:
Let me check, I noticed it getting better in the final ws